debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresConnectorTask.java

```diff
@@ -130,13 +131,18 @@ public ChangeEventSourceCoordinator<PostgresPartition, PostgresOffsetContext> st
             // Print out the server information
             SlotState slotInfo = null;
             try {
+                // todo vaibhav: how does it fail here, what happens
```

Comment:
Lets sanitize it before merging. We want to go ahead with this and later go for finer level retry.